### PR TITLE
module13 done

### PR DESCRIPTION
**Pull request by indubidably**

### CODE REVIEW

File: Homeworks/module13.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import time
```
Comment (VAlexandrov911) [inline Homeworks/module13.py:1]: Why? I didn't notice any usage

File: Homeworks/module13.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+
+# Credentials
```
Comment (VAlexandrov911) [inline Homeworks/module13.py:6]: For this task it's ok, but I just want to make a reminder for the future that it's not a good practice to store credentials in repo :)

**Review by VAlexandrov911 — APPROVED** [review]
Gj, nice and correct script, just left small comments not related to this task.
Approved

File: Homeworks/module13.py (hunk 1 above)
Comment (indubidably) [reply Homeworks/module13.py:1]: I think I just used PyCharm and overwritten an exiting .py file at the same time forgetting to erase first line

**Pull request merged**